install/kubernetes/helm/istio/charts/pilot/templates/crds.yaml

```diff
@@ -124,6 +124,20 @@ spec:
 ---
 kind: CustomResourceDefinition
 apiVersion: apiextensions.k8s.io/v1beta1
+metadata:
+  name: meshpolicies.authentication.istio.io
+spec:
+  group: authentication.istio.io
+  names:
+    kind: MeshPolicy
```

Comment:
Excuse my drive-by review: you are not the first person to try to do that, but can we be a bit more specific in naming the CRDs? The user experience is going to be about the short name, users type `kubectl get meshpolicy` to read this CR. Is this really THE mesh policy?

Reply:
This is a cherry-pick of a PR in master. @diemtvu , any idea?

Reply:
It follows the previous trend in Istio config. I was told to use just Policy and put it in authentication api group. The issue is CRD kind doesn't have *authentication*  prefix in it.